eth/config.go

```diff
@@ -87,6 +87,9 @@ type Config struct {
 	SyncMode  downloader.SyncMode
 	NoPruning bool
 
+	// Block hash whitelist
+	BlockHashWhitelist *common.BlockHashWhitelist `toml:"-"`
```

Comment:
`BlockHashWhitelist` -> `Whitelist`

Reply:
There's no reason to use a pointer here. A map can be nil.

Reply:
Done.